src/SampleViewExtension/SampleViewExtension.csproj

```diff
@@ -34,36 +34,52 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="CoreNodeModels, Version=1.1.0.2177, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\DynamoVisualProgramming.WpfUILibrary.1.1.0\lib\net45\CoreNodeModels.dll</HintPath>
+    <Reference Include="Builtin, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
@aparajit-pratap should we match the version number of this assembly to the dynamo core version? Does it not use the assembly shared info generator to set the version number?

Reply:
@mjkkirschner good catch! Will check it out.